content/documentation/pages/2-concepts/3-streams.md

```diff
@@ -19,38 +22,38 @@ Deployed stream applications communicate over a messaging middleware product.
+The middleware a stream application uses is determined by adding a specific [Spring Cloud Stream Binder](https://cloud.spring.io/spring-cloud-stream/#binder-implementations) library as a dependency to the application.
 
 ## Spring Cloud Stream
 
 For Spring developers, we suggest writing custom Stream applications by using the [Spring Cloud Stream](https://spring.io/projects/spring-cloud-stream) framework. Spring Cloud Stream lets you easily build highly scalable event-driven microservices connected with shared messaging systems.
 
 As a developer, you can focus on developing the application’s business logic while delegating the underlying API complexity and connectivity boilerplate with the message broker to Spring Cloud Stream.
 
-At a high level, streaming applications can produce or consume events through the messaging middleware.
+At a high level, streaming applications can produce, process, or consume events through the messaging middleware.
 
 ## Running Streaming Apps in the Cloud
 
 Both Cloud Foundry and Kubernetes support the concept of running long-lived applications on their platforms.
 Cloud Foundry refers to each one as a Long Running Process (LRP). On Kubernetes, you can use a deployment resource, which, in turn, manages replica sets that keep the specified number of pods to run your application.
 
-While Spring Cloud Stream can simplify your life when you write a streaming application, when a collection of independent Spring Cloud Streaming applications are deployed, you need to:
+While Spring Cloud Stream on it's own can greatly simplify your life when you write a streaming application, when a collection of independent Spring Cloud Streaming applications are manually deployed, you need to:
 
 - Configure all of the applications input and output destinations.
-- Configure the common name of a shared consumer group property to ensure there can be competing consumers on a destination.
+- Configure the common name of a shared consumer group property to ensure there is appropriate separation or cooperation of consumers on a destination.
 - Configure several properties that enable application identification and publishing of metrics information for monitoring purposes.
-- Configure the connection to your messaging middleware.
+- Configure the connection to your messaging middleware and create destinations, manage balance, scale, and partitioning.
 - Create the necessary platform resources to run the applications.
 
-When you deploy a stream with Spring Cloud Data Flow, it handles all of these configuration tasks for you and creates the necessary platform resources to run your application on the target platform.
+When you deploy a collection of streaming applications as part of a stream with Spring Cloud Data Flow, the system handles all of these configuration tasks for you and creates the necessary infrastructure resources to run your application on the target platform.
```

Comment:
> ... as part of a stream definition in  ..

Reply:
Let's remove "tasks" from "configuration tasks"; it could be confused with "task" as a primitive in SCDF.